roles/awsenv/tasks/network.yml

```diff
@@ -49,15 +50,15 @@
 
   - name: Set value of new gateway id for route table
     set_fact:
-      nat_gw_id: "{{ nat_gw.nat_gateway.nat_gateway_id }}"
+      nat_gw_id: "{{ nat_gw.nat_gateway_id }}"
```

Comment:
It changed because we defaulted to ansible's modules. Nat gateway was one that changed structure.